extensions/libraries/redcore/layouts/searchtools/default/bar.bs3.php

```diff
@@ -26,28 +26,34 @@
 
 $filters = $data['view']->filterForm->getGroup('filter');
 ?>
+
+<script type="text/javascript">
+jQuery(document).ready(function($){
+	jQuery("input[type='text']").addClass('form-control');
+});
+</script>
 <?php if (!empty($filters[$searchField])) : ?>
 	<?php if ($searchButton) : ?>
-		<label for="filter_search" class="element-invisible">
-			<?php echo JText::_('LIB_REDCORE_FILTER_SEARCH_DESC'); ?>
-		</label>
-		<div class="btn-wrapper input-append">
-			<?php echo $filters[$searchField]->input; ?>
-			<button type="submit" class="btn btn-default hasTooltip" title="<?php echo RHtml::tooltipText('JSEARCH_FILTER_SUBMIT'); ?>">
-				<i class="icon-search"></i>
-			</button>
-		</div>
-		<?php if ($filterButton) : ?>
-			<div class="btn-wrapper hidden-xs">
-				<button type="button" class="btn btn-default hasTooltip js-stools-btn-filter" title="<?php echo RHtml::tooltipText('JSEARCH_TOOLS_DESC'); ?>">
-					<?php echo JText::_('JSEARCH_TOOLS');?> <i class="caret"></i>
-				</button>
+		<section class="form-inline">
```

Comment:
I am not 100% of merging things like this because those layouts are usually not overrided by templates and it may break things in our designs. @hanshenrikchr @mshandak @Cybernary  can you please take a look here just to confirm we will not break designs with those kind of changes